backend/tracim_backend/config.py

```diff
@@ -147,6 +147,9 @@ def load_config(self) -> None:
         self._load_caldav_config()
         self._load_search_config()
         self._load_collaborative_document_edition_config()
+        import tracim_backend.applications.share.config as share_app_config
```

Comment:
Cannot be imported on top of file ? If no, think to add a NOTE about cyclic import problem.

Reply:
currently, it may work (need confirmation), but as those are import for specific app. import should be later done dynamically according to enabled app.
I prefer to have very clear point of connection between app and config to make thing easy when we will be able to make thing pluggable.

Reply:
That's a good reason. Can you add it as a note ?

Reply:
done.